### DISCUSSION TITLE
Preview page with relative resource references?

### DISCUSSION THREAD

**Discussion by carueda** [meilisearch/product#383] [Feedback & Feature Proposal]
I have a proxy-pass on my server so the meilisearch service (running on `http://localhost:7700/`) gets exposed as `https://mysite.net/meilisearch/`.

API interactions work just fine.

However, the preview page at that site fails with 404 to load resources like  `https://mysite.net/static/js/2.227a1ad8.chunk.js` apparently because of the absolute paths `/static/...` (and others).

Is there a way to make the preview page use relative references so it works as well under this scenario? Am I missing something?

Thanks in advance.  (btw, great project!)

**Comment by gmourier** [comment]
Hello @carueda 👋! Thanks for the kind word and your feedback.

Not sure to understand here, when you speak about the preview page, is it related to the mini-dashboard allowing you to do some search tests?

Which version are you using?

Thanks!

**Comment by carueda** [comment]
Thanks for the quick reply, @gmourier .

Yes, the "Mini Dashboard".

```
Server listening on:    "http://127.0.0.1:7700"
Environment:            "development"
Commit SHA:             "622c15e82558890e88da16e687d3d996085db44a"
Commit date:            "2022-01-27T10:38:41+00:00"
Package version:        "0.25.2"
```

If I open `http://127.0.0.1:7700/` directly in my browser, all works fine, and this this because (as I can see in the html source itself) the server is able to resolve all those resources with absolute references: 

```html
...
    <link rel="icon" href="/favicon.ico"/>
...
    <link rel="manifest" href="/manifest.json"/>
...
<script src="/static/js/2.227a1ad8.chunk.js"></script>
<script src="/static/js/main.8ab5ae94.chunk.js"></script>
```

My observation is whether a future version of the mini dashboard page could use relative references instead of absolute ones (the leading `/` is the problematic piece) so it can also be used remotely via a simple proxy like the one indicated. A workaround for me is to add some more proxy settings, but I thought I'd mention this here. Thanks again.

**Comment by bidoubiwa** [comment]
Hello @carueda it is indeed linked to the mini dashboard. Here is [the issue on the subject](https://github.com/meilisearch/mini-dashboard/issues/236)! Thanks a lot for you feedback :)

**Reply by gmourier** [reply]
Thanks @bidoubiwa !
